pandas/io/formats/style.py

```diff
@@ -141,12 +141,19 @@ class Styler(StylerRenderer):
         Object to define how values are displayed. See ``Styler.format``. If not given
         uses ``pandas.options.styler.format.formatter``.
 
+        .. versionadded:: 1.4.0
```

Comment:
since you are adding this to set_table_styles, do we need to add here (not a big deal but we already have a lot of options)

Reply:
for preference I prefer to. most dont import `Styler` class but those who do might use it, and there are unlikely to be many more options added here in futures so this may be the last...